jenkins/ci.suse.de/cloud-socok8s.yaml

```diff
@@ -6,3 +6,4 @@
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
       - '{name}-integration'
+      - '{name}-nightly-airship-from-rpm'
```

Comment:
Now that we have `stable/1.0` and `master`, I think we need 2 jobs

Reply:
This requires a bit of reworking. Can we do that in a followup?

Reply:
Yes. True